crates/biome_js_analyze/tests/specs/nursery/noThenProperty/invalid.js

```diff
@@ -0,0 +1,72 @@
+// `object`
+const foo = {then: 1}
+const foo = {["then"]: 1}
+const foo = {[`then`]: 1}
+const foo = {["then"]() {}}
+const foo = {[`then`]() {}}
+// const THEN = "then";const foo = {[THEN]() {}}
```

Comment:
Does anybody know how I can get the value of a variable that's referenced inside an object to handle cases like this.

Reply:
I think we should not handle this case.